### DISCUSSION TITLE
Improve handling of git packages

### DISCUSSION THREAD

**Discussion by sonicbhoc** [fosskers/aura#811] [Ideas]
So one thing that I've noticed is that I have to constantly tell Aura for each package that I don't want to view diffs for the build files. On top of this, there doesn't seem to be a way to automatically skip packages that haven't changed their build files. I'd love to see options to configure these (and if they are there, I don't know how I missed them).

Finally, I noticed that some packages fail when copying files or when the build uses cmake due to path issues. I'm not sure exactly what causes these failures.

The packages that failed include:

* ofono-git
* uavs3d-git
* bluez-tools-git

**Comment by fosskers** [comment]
Hi there, I've transferred this to an Issue. Let's chat further there. Thanks!

https://github.com/fosskers/aura/issues/812
